src/mocks/handlers/historyHandlers.ts

```diff
@@ -41,7 +41,8 @@ export function buildHistory(workNumber?: string): HistoryFieldsFragment {
     __typename: 'History',
     samples: [sample],
     labware: [sourceLabware, destinationLabware].map(buildLabwareFragment),
-    entries
+    entries,
+    samplePositionResults: []
```

Comment:
This mocking is not correct as the core side doesn't support this

Reply:
The core code, is been updated accordingly, you can have a look here [#237](https://github.com/sanger/stan-core/pull/237)